source/EvoPlanet/evoplanet.client/src/app/components/planet-system/planet-system.component.ts

```diff
@@ -12,16 +13,22 @@ import { Planet, SolarSystem } from '../../types';
 
 export class PlanetSystemComponent implements OnInit {
   planetSystem: Array<SolarSystem> = [];;
+  router = inject(Router);
```

Comment:
I think the usage of `inject` here makes the code inconsistent. The `private solarSystemService: SolarSystemService` part in the constructor does injection too. Why did you use `inject` instead of constructor private parameters?

I'm not saying one is necessarily better than the other, but it's best to keep things consistent in the code, and when you need to diverge make a clear rationale about it. It's also best to talk with your team about it.

Reply:
I just wanted to be quick and didn't notice the inconsistency.